builder/.dockerignore

```diff
@@ -0,0 +1,13 @@
+# Exclude everything
```

Comment:
I use whitelist everywhere because in most case I just need the source `src/` and `package.json` rather than ignoring dozens of files and then missing some when we had news. And now `nbuild` is not here anymore since it's pulled from docker hub